src/FlapperUniV2SwapOnly.sol

```diff
@@ -33,6 +33,7 @@ interface SpotterLike {
 
 interface GemLike {
     function decimals() external view returns (uint8);
+    function transfer(address, uint256) external;
```

Comment:
Can't mark in the line, but we should remove `move` from `VatLike`.

Reply:
And `exit` from `DaiJoinLike`

Reply:
Done